README.md

```diff
@@ -1,226 +1,188 @@
-![Kin Logo](kin.png)  ![Stelalr Logo](stellar.png)
+![Kin Logo](kin.png)
+
+# KIN Python SDK for Kin Blockchain
 
-# KIN Python SDK for Stellar Blockchain
-[![Build Status](https://travis-ci.org/kinecosystem/kin-core-python.svg?branch=master)](https://travis-ci.org/kinecosystem/kin-core-python) [![Coverage Status](https://codecov.io/gh/kinecosystem/kin-core-python/branch/master/graph/badge.svg)](https://codecov.io/gh/kinecosystem/kin-core-python)
 
 ## Disclaimer
 
 The SDK is still in beta. No warranties are given, use on your own discretion.
 
 ## Requirements.
 
-Make sure you have Python 2 >=2.7.9.
+Python 2 > 2.7.9 / 3 > 3.4
 
 ## Installation 
 
 ```bash
 pip install git+https://github.com/kinecosystem/kin-core-python.git
+# TODO: add to pypi after PR is done
```

Comment:
TODO should be in Jira...

Reply:
I will add it to JIRA, but its a reminder that a normal ```pip install kin-core``` will be available later